### PR TITLE
Auto-push data repo when changes are committed.

### PR DESCRIPTION
**Pull request by srfraser**
Addresses issue #16

### CODE REVIEW

**Comment by Callek** [comment]
@srfraser @lundjordan I for one like the ability to push the data without having to cd to the data repo, however I frequently do a handful of things and then want to push, including potentially wanting to abort what I did.  (git reset --hard HEAD^1)

In rw1 I had an issue where it would clear out a larger swath of data intermittently so I had to reset then re-do my command to fix, so this could cause problems if that happens again here for some reason.

I'd love if we could make this a config entry, possibly even a global arg --no-push // --push, thoughts?

**Comment by srfraser** [comment]
It may be simplest to add a 'git push' subcommand, the name of which we can bikeshed

File: releasewarrior/wiki_data.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ def write_and_commit(data, data_path, wiki_path, commit_msg, logger, config, wik
     logger.debug(wiki_path)
     logger.debug(corsica_path)
     commit([data_path, wiki_path, corsica_path], commit_msg, logger, config)
+    push(logger, config)
```
Comment (lundjordan) [inline releasewarrior/wiki_data.py]: Like @Callek was suggesting, if we condition this off an option or make a separate subcommand, I'm all for it.

I think this is a really nice feature and skips some manual pain.

fwiw - right now I have the following aliases that act as subcommands so I don't need to cd and type long git commands:

alias rwl='git -C /Users/jlund/devel/mozilla/repos/releasewarrior-data pull upstream master'
alias rwd='git -C /Users/jlund/devel/mozilla/repos/releasewarrior-data diff upstream/master'
alias rwp='git -C /Users/jlund/devel/mozilla/repos/releasewarrior-data push upstream master'

Comment (lundjordan) [reply releasewarrior/wiki_data.py]: @srfraser @Callek how would we like to follow up here? Condition this call off a config option, a cli option, or no condition at all?

Comment (Callek) [reply releasewarrior/wiki_data.py]: @lundjordan I suggested my fix by a WIP PR in #42

Comment (srfraser) [reply releasewarrior/wiki_data.py]: I'm happy with #42, and we anc always add more options, or a 'pass all this to git and add -C' option if it gets that far

Comment (lundjordan) [reply releasewarrior/wiki_data.py]: okay closing this for now due to last two comments

**Pull request closed without merging**